src/main/java/org/example/homework11/CollectionsOfSymbols.java

```diff
@@ -0,0 +1,18 @@
+package org.example.homework11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CollectionsOfSymbols {
+    public Map<String, Integer> mapSymbols(String sentence) {
+        Map<String, Integer> lettersCount = new HashMap<>();
+        char[] symbolsArray = sentence.toCharArray();
+        for (char symbol : symbolsArray) {
+            if (String.valueOf(symbol).matches("[\\s\\p{P}]")) {
```

Comment:
```suggestion
           if (!Character.isLetter(symbol)) {
```
this looks better and you get rid of redundant regular expression